README.md

```diff
@@ -138,12 +138,12 @@ With that setup, you could do the following in the terminal:
 - **Compile Sass:** `npx gulp compile` or `npx gulp`
 - **Watch for changes and recompile:** `npx gulp watch`
 - **Initialize a new project:** `npx gulp init`
-- **Update USWDS static assets and recompile:** `npx gulp update`
+- **Update USWDS static assets and recompile:** `npx gulp updateUswds`
 
 ### Usage tips
 
 - **Use `init` only once.** The `init` task is meant for initializing the design system on a project. Since it will overwrite project files (like settings files and the Sass entry point), use it sparingly and don't use it for updating the design system on a project, or at any point after you've customized your settings files.
-- **Update USWDS assets with `copyAssets`.** Don't update assets with `init`, use the `copyAssets` task. This task updates static assets (like images, fonts, and compiled JavaScript) only and you don't risk clobbering your customizations.
+- **Staying up-to-date with USWDS.** When updating your project's version of USWDS; code won't be copied over until you run the `npx gulp updateUswds` command. You can see all of the required changes available on the [USWDS releases](https://github.com/uswds/uswds/releases) page. Alternatively, you can run the `copyAssets` task. This task updates static assets (like images, fonts, and compiled JavaScript) only and you don't risk clobbering your customizations.
```

Comment:
This is a bit more verbose, but made a few of suggestions to try to improve clarity. My main concerns were:
1. It wasn't clear to me what code would be copied over. 
2. It wasn't clear to me what "required changes" meant. 
3. Because of the structure, it read to me like `updateUswds` and `copyAssets` were doing different things, but it looks like they are the same, only `updateUswds` also includes a compile task. 

Take or leave what you will!

```suggestion
- **Stay up-to-date with USWDS with `updateUswds`.** When updating your project's version of USWDS, run the `npx gulp updateUswds` command to copy any updated USWDS assets into your project. Alternatively, you can run the `copyAssets` task. These tasks update USWDS static assets (like images, fonts, and compiled JavaScript) and will not override any of your project customizations. These actions might introduce breaking changes, so be sure to check out the related [USWDS releases notes](https://github.com/uswds/uswds/releases) for any additional tasks that must be completed when updating to a new version of USWDS. 
```

Reply:
Thanks for the suggestion, much clearer.